ocpp/exceptions.py

```diff
@@ -24,6 +24,10 @@ def __repr__(self):
         return f"<{self.__class__.__name__} - description={self.description},"\
                f" details={self.details}>"
 
+    def __str__(self):
+        return f"<{self.__class__.__name__} - description={self.description},"\
+               f" details={self.details}>"
```

Comment:
```suggestion
        return f"{self.__class__.__name__}: {self.description},"\
               f" {self.details}"
```

In my opinion, because this method would most likely be used to show information to the client (not for debugging purposes, as `__repr__` should be used for that), `__str__` shouldn't contain the `<` and `>` beginning and ending. Also, I'd not write `description=` and `details=` but directly print them.

Although nitpicky, I'd also say using `:` instead of `-` keeps the format more similar to how exceptions are printed by default. That's debatable, though.

Reply:
yeah, looks better actually. I was just not sure if the exceptions would include in the future details and descriptions.